Dockerfile

```diff
@@ -1,31 +1,5 @@
-FROM node:9
-MAINTAINER Adam Frank <afrank@mozilla.com>
+FROM etherpad/etherpad:1.8.14
```

Comment:
Dockerfile looking a lot easier to understand

Reply:
had to add jq & curl for the pads cleanup script, but yeah, its a bit simpler.